src/sage/schemes/toric/weierstrass.py

```diff
@@ -220,10 +220,10 @@ def j_invariant(polynomial, variables=None):
         ...
         ValueError: curve is singular and has no well-defined j-invariant
     """
-    (f, g) = WeierstrassForm(polynomial, variables)
-    disc = 4*f**3+27*g**2
+    f, g = WeierstrassForm(polynomial, variables)
+    disc = 4 * f**3 + 27 * g**2
```

Comment:
```suggestion
    disc = 4*f**3 + 27*g**2
```
top-level only please